girder/utility/path.py

```diff
@@ -21,20 +21,13 @@
 
 import re
 from ..constants import AccessType
-from ..models.model_base import AccessException, GirderException, ValidationException
+from ..exceptions import AccessException, GirderException, ValidationException
+from ..exceptions import ResourcePathNotFound as NotFoundException
```

Comment:
There are still a number of statements that `raise NotFoundException` in this file. These should all be changed to `raise ResourcePathNotFound`, so that viewers of the code aren't confused about what the canonical name of the exception is.

I expect that you should then have a `from ..exceptions import ResourcePathNotFound` statement and a global `NotFoundException = ResourcePathNotFound` statement at the top, with a Flake8 ignore directive on the latter, since it shouldn't be used, but will just make the symbol available in this module's namespace for compatibility.

To be clear, I think using the `from ... import ... as ...` would be fine if it's only used for symbol definition compatibility. If the symbols are actually being referenced within the code, I think we need to consistently use the new name,

Reply:
Agreed.